target/linux/ath79/dts/ar9344_devolo_dlan2-2400-ac.dts

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "Devolo Magic 2 Wifi";
+	compatible = "devolo,dlan2-2400-ac", "qca,ar9344";
```

Comment:
Okay, I see that you changed everything consistently to "devolo,dlan2-2400-ac".
However, as I tried to point out in the initial comment, if you call the device "Magic 2 Wifi", you should use the same "name" for the identifier. So, please change "devolo,dlan2-2400-ac" to "devolo,magic-2-wifi" everywhere.